interm_notebook_5_classify.ipynb

```diff
@@ -0,0 +1,1175 @@
+{
```

Comment:
<p>You had mentioned adding a section here about overfitting -- that so long as the testing/validation accuracies are both going up, we can be reasonably confident that we haven't overfit. </p><p> </p><p>I would add that here, before the reader launches into the next section. It helps to motivate these plots</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>